stream-registry-data/src/main/java/com/expediagroup/streamplatform/streamregistry/ModelToData.java

```diff
@@ -0,0 +1,146 @@
+/**
+ * Copyright (C) 2018-2020 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.expediagroup.streamplatform.streamregistry;
+
+import static com.expediagroup.streamplatform.streamregistry.data.ObjectNodeMapper.serialise;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import com.expediagroup.streamplatform.streamregistry.data.Consumer;
+import com.expediagroup.streamplatform.streamregistry.data.ConsumerBinding;
+import com.expediagroup.streamplatform.streamregistry.data.Domain;
+import com.expediagroup.streamplatform.streamregistry.data.Infrastructure;
+import com.expediagroup.streamplatform.streamregistry.data.Producer;
+import com.expediagroup.streamplatform.streamregistry.data.ProducerBinding;
+import com.expediagroup.streamplatform.streamregistry.data.Schema;
+import com.expediagroup.streamplatform.streamregistry.data.Specification;
+import com.expediagroup.streamplatform.streamregistry.data.Status;
+import com.expediagroup.streamplatform.streamregistry.data.Stream;
+import com.expediagroup.streamplatform.streamregistry.data.StreamBinding;
+import com.expediagroup.streamplatform.streamregistry.data.Tag;
+import com.expediagroup.streamplatform.streamregistry.data.Zone;
+import com.expediagroup.streamplatform.streamregistry.data.keys.ConsumerBindingKey;
+import com.expediagroup.streamplatform.streamregistry.data.keys.ConsumerKey;
+import com.expediagroup.streamplatform.streamregistry.data.keys.DomainKey;
+import com.expediagroup.streamplatform.streamregistry.data.keys.InfrastructureKey;
+import com.expediagroup.streamplatform.streamregistry.data.keys.ProducerBindingKey;
+import com.expediagroup.streamplatform.streamregistry.data.keys.ProducerKey;
+import com.expediagroup.streamplatform.streamregistry.data.keys.SchemaKey;
+import com.expediagroup.streamplatform.streamregistry.data.keys.StreamBindingKey;
+import com.expediagroup.streamplatform.streamregistry.data.keys.StreamKey;
+import com.expediagroup.streamplatform.streamregistry.data.keys.ZoneKey;
+
+public class ModelToData {
+
+  public static ConsumerKey convertToData(com.expediagroup.streamplatform.streamregistry.model.keys.ConsumerKey in) {
+    return new ConsumerKey(in.getStreamDomain(), in.getStreamName(), in.getStreamVersion(), in.getZone(), in.getName());
```

Comment:
[bull](https://github.com/HotelsDotCom/bull) could do most of this automatically, you'd only need to provide handlers for the config/status serde.

It's already being dependency managed as was being used previously https://github.com/ExpediaGroup/stream-registry/blob/410a45bc01cbacf1c93aab60324f8f066efa92e8/pom.xml#L135

Reply:
Done.